mono/mini/aot-compiler.c

```diff
@@ -6585,7 +6587,7 @@ emit_trampolines (MonoAotCompile *acfg)
 	int tramp_type;
 #endif
 
-	if (!mono_aot_mode_is_full (&acfg->aot_opts) || acfg->aot_opts.llvm_only)
+	if (!HAS_AOT_FEATURE (&acfg->aot_opts, TRAMPOLINE) || acfg->aot_opts.llvm_only)
 		return;
```

Comment:
Can't we use bools here like in MonoCompile, i.e. acfg->enable_trampolines or something ?